android/app/build.gradle

```diff
@@ -37,18 +37,7 @@ android {
     versionCode 115
     versionName '2.12.0'
     // END VERSIONS
-    ndk {
-      // We use different `abiFilters` for distributing
-      // to be able to bundle old, incompatible SDKs.
 
-      // WHEN_DISTRIBUTING_REMOVE_FROM_HERE
-      abiFilters 'armeabi-v7a', 'x86'
-      // WHEN_DISTRIBUTING_REMOVE_TO_HERE
-
-      /* UNCOMMENT WHEN DISTRIBUTING
-      abiFilters 'armeabi-v7a', 'x86', 'arm64-v8a', 'x86_64'
```

Comment:
Are you sure we can just remove this whole section? Shouldn't we keep that single line above to have only those 4 abis enabled?

Reply:
There are only binaries for those 4 platforms anywhere in our project so it should be fine

Reply:
So there is no possiblity of a developer specifying another set of libraries and trying to build an app with abi filter `'myabi-v10x'` which will build successfully, but then crash in runtime?

Reply:
it should fail during the build